plugin/pkg/admission/runtimeclass/admission.go

```diff
@@ -193,6 +231,28 @@ func validateOverhead(a admission.Attributes, pod *api.Pod, runtimeClass *v1beta
 	return nil
 }
 
+func validateScheduling(a admission.Attributes, pod *api.Pod, runtimeClass *v1beta1.RuntimeClass) (err error) {
+	fmt.Println(runtimeClass)
+	if runtimeClass != nil && runtimeClass.Scheduling != nil {
```

Comment:
nit: short circuit this case. Same elsewhere.
```suggestion
	if runtimeClass == nil || runtimeClass.Scheduling == nil {
	  return nil
	}
```